cli/src/main/java/org/apache/falcon/cli/FalconEntityCLI.java

```diff
@@ -329,13 +362,13 @@ public void entityCommand(CommandLine commandLine, FalconClient client) throws F
         OUT.get().println(result);
     }
 
-    private void validateColo(Set<String> optionsList) throws FalconCLIException {
+    private void validateColo(Set<String> optionsList) {
```

Comment:
I was clearly getting `Redundant throws` on both local and jenkins. Some of the builds have failed with that. I could prove it to you but the job logs would have been lost.